### DISCUSSION TITLE
highlight guisp 不起作用

### DISCUSSION THREAD

**Discussion by allworldg** [nshen/learn-neovim-lua#160] [Q&A]
我想报错代码的下划线呈现不同的颜色。所以我尝试使用`highlight DiagnosticUnderlineError gui = underline guifg = #F03D22 guibg = White guisp=Grey` 。结果是报错信息和下划线全变成红色。
![image](https://user-images.githubusercontent.com/48782874/223320620-b9c332b7-6b0a-49e1-ad6a-b0441d40f097.png)
有看到可能terminal不支持underCurl（卷曲）的介绍，但是我的underline起码可以显示。

environment：
Windows Terminal+wsl2+neovim ( 0.8.3 ) .

尝试替换成Wezterm,alacritty,wsltty等终端，都没有生效。请问大家有解决方法吗？ 
谢谢。

**Comment by nshen** [comment]
不太了解这块，对这个没有太高要求。帮顶
